### ISSUE TITLE
Change navbar buttons to be actual buttons to improve accessibility

### ISSUE THREAD

**Issue by JRIngram** [JRIngram/leitner#32]
# Bug Report
## Description 

The navbar "buttons" aren't actually buttons but <p> tags. These should be changed to <buttons> to improve accessibility.
